pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -109,16 +111,13 @@ func (r *ReconcileJaeger) Reconcile(request reconcile.Request) (reconcile.Result
 		return reconcile.Result{}, err
 	}
 
-	logFields.Info("Configured Jaeger instance")
```

Comment:
Do you want to remove it? We could indicate the changes on line 119

Reply:
Yeah, I wanted to remove it. As someone managing the cluster, I'm not sure this would be useful at all in the day-to-day. It's certainly useful to get a positive confirmation in the beginning, but considering that there will be one change every 5 seconds (due to the metrics on #248), this would cause too much noise in the logs.